src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,114 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private final int capacity = 10;
+    private final int changeStep = 1;
+    private final int zeroIndex = 0;
+    private T[] userArray;
```

Comment:
```suggestion
    private T[] values;
```
```suggestion
    private T[] elements;
```
```suggestion
    private T[] data;
```
don't include data type in the variable's name